doc/api/repl.md

```diff
@@ -147,6 +147,13 @@ global or scoped variable, the input `fs` will be evaluated on-demand as
 > fs.createReadStream('./some/file');
 ```
 
+Subpaths Node.js core modules are exposed with a underscore,
+for example, `stream/consumers` will become `stream_consumers`.
+
+```console
+> stream_consumers.text(process.stdin);
+```
```

Comment:
I'm not sure if this is really the approach we should take. I'd prefer these submodules to be just consistently accessible via properties (e.g. `stream.consumers`)

Reply:
I like the idea but what if this property already exists?

Reply:
> what if this property already exists?

Is there such a case?

Reply:
For the moment, no.

Reply:
You are right. :+1:

Reply:
I would really not add things to the REPL that aren’t also in existence without the REPL; i.e. `stream.consumers` being accessible but `require('stream').consumers` not being accessible would be bad, imo.

Reply:
So what would be the best solution? Export each submodule in its parent module (like `fs.promises`, etc.)?

Reply:
> stream.consumers being accessible but require('stream').consumers not being accessible would be bad, imo.

Fwiw, that's not what I was suggesting. Just expose `consumers` as a property of `stream`.

Reply:
Ah okay what about other modules? For example `stream/web`/`stream.web`? Same for all modules-submodules?

Reply:
ping @jasnell